### PR TITLE
Switching from polyfill.io to cloudflare

### PR DESCRIPTION
**Pull request by oomIRL**
## Summary
Quick Summary Here
Switching out polyfill.io due to supply chain vulnerability concerns.

## Notable Changes 
-
-
-

### CODE REVIEW

**Review by MonicaG — APPROVED** [review]
LGTM

**Comment by oomIRL** [comment]
@deploy-tron help

**Comment by oomIRL** [comment]
@deploy-tron help

**Comment by oomIRL** [comment]
@deploy-tron deploy web to dev

**Review by sheaphillips — APPROVED** [review]
looks good.

**Comment by sheaphillips** [comment]
@oomIRL  looks like there's some Actions issues to look into.

**Comment by oomIRL** [comment]
@deploy-tron deploy web to dev

**Comment by oomIRL** [comment]
@deploy-tron deploy web to dev

**Comment by oomIRL** [comment]
@deploy-tron deploy web to dev

File: web/topicRegistry/data-and-apis.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
           "owner": "bcgov",
           "title": "BC Government API Registry",
           "description": "The Province of British Columbia API Registry",
-          "image": "https://developer.gov.bc.ca/images/gov-logo-static.png"
+          "image": "https://classic.developer.gov.bc.ca/images/gov-logo-static.png"
```
Comment (MonicaG) [inline web/topicRegistry/data-and-apis.json:24]: ohhh, it was the image location changed, good catch!

Comment (oomIRL) [reply web/topicRegistry/data-and-apis.json:24]: Thanks! Took some digging around :)

**Comment by oomIRL** [comment]
@deploy-tron deploy web to staging

**Comment by oomIRL** [comment]
@deploy-tron deploy web to dev

**Comment by oomIRL** [comment]
@deploy-tron deploy web to staging

**Comment by oomIRL** [comment]
@deploy-tron deploy web to test

**Comment by ghost** [comment]
### Deployment Created
Deployment 1622668550 to staging created :) 

The deployment has the following payload which you can integrate with your ci/cd service.
```json
{"microservice":"web","pr":1818,"user":"oomIRL"}
```

**Comment by oomIRL** [comment]
@deploy-tron deploy web to prod

**Comment by oomIRL** [comment]
@deploy-tron deploy web to prod

**Pull request merged**